ChromeExtension/manifest.json

```diff
@@ -7,6 +7,7 @@
     },
     "page_action": {
       "default_popup": "popup.html",
+      "default_popup": "TipPopUp.html",
       "default_icon": {
         "260": "images/sustainaBrowseLogo.png"
       }
```

Comment:
I like this. I actually didn't think of just writing my own json page. I think it might be a good idea for what I am trying to do.